velox/functions/prestosql/DecimalVectorFunctions.cpp

```diff
@@ -102,9 +102,13 @@ template <
     typename R /* Result Type */,
     typename A /* Argument */,
     typename Operation /* Arithmetic operation */>
-class DecimalUnaryBaseFunction : public exec::VectorFunction {
+class DecimalUnaryFunction : public exec::VectorFunction {
  public:
-  explicit DecimalUnaryBaseFunction(uint8_t aRescale) : aRescale_(aRescale) {}
+  explicit DecimalUnaryFunction(
+      uint8_t aPrecision,
+      uint8_t aRescale,
+      int param = INT_MAX)
```

Comment:
Do we need this default?

Reply:
Removed.

Reply:
Still there?

Reply:
I think we should make this `std::optional<int> param` for clarity.